source/domain/pdf_model.py

```diff
@@ -0,0 +1,54 @@
+from fpdf import FPDF
+
+from source.domain.utils import extract_files_to_list
+
+
+class PDF(FPDF):
+    def header(self):
+        self.image("resources/icons/hexagon.png", 10, 8, 25)
+
+        self.set_font("Helvetica", 'B', 20)
+
+        self.cell(0, 10, 'Graph Filter', border=False, ln=1, align='C')
+
+        self.set_font("Helvetica", 'B', 14)
+        self.cell(0, 10, "Information about the filtering", align='C')
+        self.ln(20)
+
+    def footer(self):
+        self.set_y(-15)
+
+        self.cell(0, 10, f'Page {self.page_no()}/{{nb}}', align='C')
+
+    def information_about_filtering(self, name, method, equations, conditions, input_files):
+        self.add_font('DejaVuSans', '', 'resources/fonts/DejaVuSans.ttf')
+        self.set_font('DejaVuSans')
+        self.cell(0, 10, f"Name: {name}", ln=True)
+
+        self.cell(0, 10, f"Methods: {method}", ln=True)
+
+        self.multi_cell(0, 10, f"Conditions: {conditions}", ln=True)
+
+        self.cell(0, 10, f"Equations: {equations}", ln=True)
+
+        self.cell(0, 10, f"Files: ", ln=True)
```

Comment:
Insert the field "Project Description". Write only in case this field is not empty.

Reply:
Done!
I prefer to not make verifications when its empty, if the description is empty I just show the camp empty